packages/headless/src/utils/resolveTemplate.ts

```diff
@@ -0,0 +1,39 @@
+import { UriInfo } from '../types';
+import type { Template } from '../components/TemplateLoader';
+
+export function resolveTemplate(
+  pageInfo: UriInfo | undefined,
+): null | Template {
+  /**
+   * List out files in main project using Webpack.
+   */
+  let context;
+
+  try {
+    context = require.context('theme', false, /.*/);
+  } catch (e) {
+    console.warn(
```

Comment:
Is it worth allowing console warnings and errors to suppress this ESLint warning?

https://eslint.org/docs/rules/no-console#options

Reply:
Great callout! I'll make that adjustment as we'll likely continue raising warnings/errors.